06-setup-backup-restore/trilio.md

```diff
@@ -261,25 +262,35 @@ If the output looks like above, you installed `TVK` successfully. Next, you will
 By default, when installing `TVK` via `Helm`, there is no `Free Trial` license installed automatically. You can always go to the `Trilio` website and generate a new [license](https://www.trilio.io/plans) for your cluster that suits your needs (for example, you can pick the `basic license` type that lets you run `TrilioVault` indefinetly if your `cluster capacity` doesn't exceed `10 nodes`). A free trial license lets you run `TVK` for `one month` on `unlimited` cluster nodes.
 
 **Notes:**
-- **TrilioVault is free of charge for Kubernetes clusters with up to 10000 nodes for DigitalOcean users.**
+- **TrilioVault is free of charge for Kubernetes clusters with up to 100000 nodes for DigitalOcean users. User can follow below steps to create a license specifically created for DO users**
```

Comment:
The mention of TVK license being available free to DO users is important. So to avoid user of `User` multiple times, I have formulated it as below:
```
**TrilioVault is free of charge for Kubernetes clusters with up to 100000 nodes for DigitalOcean users. They can follow below steps to create a special license available for DO customers only**
```
Let me know if this looks good to you.